docs/guide.mdx

```diff
@@ -61,10 +61,10 @@ If an access token isn't provided, the following rate limits apply:
 The Wave does not support the use of sha256 digest in the image name, e.g. `ubuntu@sha256:3235...ce8f`, when using
 the augmentation process to extend container images.
 
-In order to reference a container via sha256 digest in the image name with Wave you will need to *freeze* image mode
-that will force the creation of a new container image using the container you have specified as base image.
+In order to reference a container via this sha256 digest tag, you will need to use the _freeze_ image mode.
+This will force the creation of a new container image, using the container you have specified as base image.
```

Comment:
```suggestion
This will force the creation of a new container image, using the container you have specified as base image
and push it to a 3rd party container registry.
```